config/param_types.go

```diff
@@ -140,6 +140,20 @@ func (p *SecondsParam) Parse(raw string) (result interface{}, err error) {
 	return
 }
 
+type MillisParam struct {
+	Metadata
+}
+
+func (p *MillisParam) Parse(raw string) (result interface{}, err error) {
+	millis, err := strconv.ParseFloat(raw, 64)
+	if err != nil {
+		err = p.parseFailed(raw, "invalid float")
+		return
+	}
+	result = time.Duration(millis * float64(time.Millisecond))
```

Comment:
Yes, it is.  I want to allow the input to be a float, so you could say `1.2` (milliseconds).  Go's casting rules are very strict so you can't do `float * duration` (which is really an int64 number of nanoseconds) without a cast.  Then I need to cast back to `time.Duration` at the end.

Casting millis to `time.Duration` first then multiplying doesn't work because that would lose precision.